README.md

```diff
@@ -0,0 +1,173 @@
+# Photon Ranch Calendar
+
+This repository manages the backend service for the calendar at www.photonranch.org. Communication between the web interface and calendar database operations at AWS occur here.
+
+This is a Serverless API deployed by the Serverless Framework, which creates and runs the Python functions in `handler.py` in AWS Lambda.
+
+![Photon Ranch Calendar interface](images/calendar-screenshot.PNG)
+
+## Description
+
+The Photon Ranch calendar allows authenticated users to create and manage reservations for real time and automated observing sessions. Each site page has a calendar tab. Users can request to:
+
+- Add a new reservation to the calendar
+- Modify an existing reservation
+- Add or remove a [project](https://github.com/LCOGT/photonranch-projects) to a reservation
+- Delete a reservation from the calendar
+
+![Reserving a real-time session on the Photon Ranch Calendar](images/calendar-create-reservation.PNG)
+
+## Dependencies
+
+This application currently runs under Python 3.9. Serverless requirements for deployment are listed in `package.json`. A list of Python dependencies, which the `serverless-python-requirements` plugin zips for the Lambda environment, can be found in `requirements.txt`.
+
+To update npm dependencies, run `npm update`. This will update the package versions in `package-lock.json`.
+
+## Local Development
+
+Clone the repository to your local machine:
+
+```
+git clone https://github.com/LCOGT/photonranch-calendar.git
+cd photonranch-calendar
+```
+
+### Requirements
+
+You will need the [Serverless Framework](https://www.serverless.com/framework/docs/getting-started) installed locally for development. For manual deployment to AWS as well as for updating dependencies, you will need to install [Node](https://nodejs.org/en/), [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), and [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), configuring with your own AWS credentials.
+
+### Deployment
+
+Changes pushed to the test, dev, and main branches are automatically deployed to the corresponding test, dev, and production stages with Github Actions. For manual deployment on your local machine, you'll need to fill out the `public_key` and `secrets.json` with the required information. Then, run:
+
+```
+npm install
+serverless plugin install --name serverless-python-requirements
+serverless deploy --stage {stage}
+```
```

Comment:
I would separate these steps.`npm install` and the plugin installations only need to be run once. After that, the user will only need to worry about `serverless deploy...`.

Reply:
Updated this in the readme